sites/skeleton.dev/src/routes/(inner)/elements/tables/+page.svelte

```diff
@@ -57,7 +60,7 @@
 						<tbody>
 							{#each tableArr as row}
 								<tr>
-									<td>{row.name}</td>
+									<td>{row.name} - {lorem}</td>
```

Comment:
The lorem ipsum is pretty long and especially on smaller screens it causes a lot of sideways-scrolling to be necessary to reach the other side. Maybe the lorem ipsum could be shortened a bit?
![Screen Shot 2023-12-20 at 23 17 11](https://github.com/skeletonlabs/skeleton/assets/30119708/29fa7ac3-ce9b-4c13-8247-9bec2e2a2df5)

Reply:
Good catch, it was only meant for testing. Whoops!